src/MainWindow.vala

```diff
@@ -306,6 +330,21 @@ public class AppCenter.MainWindow : Hdy.ApplicationWindow {
         search_entry.text = term;
     }
 
+    public void send_installed_toast (AppCenterCore.Package package) {
+        last_installed_package = package;
+
+        // Show a toast if
+        // no package is selected --> we are not on an app page
+        // or
+        // a package is selected --> we are on an app page
+        // but the app's name is not equal to the package's name that should be installed
```

Comment:
I think this conveys the same information more concisely, without just repeating the code as well. :smile: 

```suggestion
        // Only show a toast when we're not on the installed app's page, i.e if
        // no package is selected (we are not on an app page), or a package is 
        // selected but it's not the app we're installing.
```